lib/worker_tools/recorder.rb

```diff
@@ -23,38 +23,48 @@ def with_wrapper_logger(&block)
     end
 
     def record_fail(error)
-      record "ID #{model.id} - Error"
+      # record "ID #{model.id} - Error"
```

Comment:
please remove the comment

replace `record "ID #{model.id} - Error"` with `add_log("ID #{model.id", :error)`